onnxruntime/hosting/executor.cc

```diff
@@ -93,31 +101,32 @@ protobufutil::Status Executor::Predict(const std::string& model_name, const std:
   }
   std::vector<onnxruntime::MLValue> outputs(output_names.size());
 
-  // Run()!
+  // Run
   OrtRunOptions run_options{};
-  run_options.run_log_verbosity_level = 4;  // TODO: respect user selected log level
-  run_options.run_tag = request_id;
+  run_options.run_tag = request_id_;
+
+  auto status = env_->session->Run(run_options, name_ml_value_map, output_names, &outputs);
 
-  status = env_->GetSession()->Run(run_options, name_ml_value_map, output_names, &outputs);
   if (!status.IsOK()) {
-    LOGS(*logger, ERROR) << "Run() FAILED!"
-                         << " Error code: " << status.Code()
-                         << ". Error Message: " << status.ErrorMessage();
-    return GenerateProtoBufStatus(status, "Run() FAILED: " + status.ErrorMessage());
+    LOGS(*logger, ERROR) << "Run() failed." << ". Error Message: " << status.ToString();
+    return GenerateProtobufStatus(status, "Run() failed: " + status.ErrorMessage());
   }
 
   // Build the response
   for (size_t i = 0; i < outputs.size(); ++i) {
     onnx::TensorProto output_tensor{};
-    status = MLValueToTensorProto(outputs[i], using_raw_data, std::move(logger), output_tensor);
+    status = MLValueToTensorProto(outputs[i], using_raw_data, logger, output_tensor);
     if (!status.IsOK()) {
-      LOGS(*logger, ERROR) << "MLValueToTensorProto() FAILED! Output name: " << output_names[i]
-                           << " Error code: " << status.Code()
-                           << ". Error Message: " << status.ErrorMessage();
-      return GenerateProtoBufStatus(status, "MLValueToTensorProto() FAILED: " + status.ErrorMessage());
+      LOGS(*logger, ERROR) << "MLValueToTensorProto() failed. Output name: " << output_names[i] << ". Error Message: " << status.ToString();
+      return GenerateProtobufStatus(status, "MLValueToTensorProto() failed: " + status.ErrorMessage());
     }
 
-    response.mutable_outputs()->insert({output_names[i], output_tensor});
+    auto insertion_result = response.mutable_outputs()->insert({output_names[i], output_tensor});
+
+    if (!insertion_result.second) {
+      LOGS(*logger, ERROR) << "SetNameMLValueMap() failed. Input name: " << output_names[i] << " Trying to overwrite existing input value";
+      return protobufutil::Status(protobufutil::error::Code::INVALID_ARGUMENT, "SetNameMLValueMap() failed: Cannot have two inputs with the same name");
```

Comment:
"inputs" --> "outputs"

Reply:
Changed all of them. Lazy copy pasting :3